docs/source/release_notes.rst

```diff
@@ -4,6 +4,211 @@
+access to the model's |Operators| is possible.
+
+Our implementation internally wraps `PyTorch <https://pytorch.org>`_ for the training and evaluation of
+the neural networks. No knowledge of PyTorch or neural networks is required to apply the method.
+
+
+New system analysis and linear algebra algorithms
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+So far, computing poles of an |LTIModel| was only supported by its
+:meth:`~pymor.models.iosys.LTIModel.poles` method which uses a dense eigenvalue
+solver and converts the operators to dense matrices.
+The new version adds the :meth:`~pymor.algorithms.eigs.eigs` method
+(`[#880] <https://github.com/pymor/pymor/pull/880>`_) for computing a few
+smallest/largest eigenvalues (similar to `scipy.sparse.linalg.eigs`, except
+working for generic operators).
+Furthermore, a new addition is the :meth:`~pymor.algorithms.samdp.samdp` method
+(`[#834] <https://github.com/pymor/pymor/pull/834>`_) which takes into account
+the input and output operators of an |LTIModel| to compute its dominant poles.
+
+
+Improved parameter handling
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+While pyMOR always had a powerful and flexible system for handling |parameters|,
+understanding this system was often a challenge for pyMOR newcomers. Therefore,
+we have completely overhauled parameter handling in pyMOR, removing some unneeded
+complexities and making the nomenclature more straightforward. In particular:
+
+- The `Parameter` class has been renamed to :class:`~pymor.parameters.base.Mu`.
+  `ParameterType` has been renamed to |Parameters|. The items of a |Parameters|
+  dict are the individual *parameters* of the corresponding |ParametricObject|.
+  The items of a :class:`~pymor.parameters.base.Mu` dict are the associated
+  *parameter values*.
+- All parameters are now one-dimensional NumPy arrays.
+- Instead of manually calling `build_parameter_type` in `__init__`, the |parameters|
+  of a |ParametricObject| are now automatically inferred from the object's `__init__`
+  arguments. The process can be customized using the new `parameters_own` and
+  `parameters_internal` properties.
+- `CubicParameterSpace` was renamed to |ParameterSpace| and is created using
+  `parametric_object.parameters.space(ranges)`.
+
+Further details can be found in `[#923] <https://github.com/pymor/pymor/pull/923>`_.
+Also see `[#949] <https://github.com/pymor/pymor/pull/949>`_ and
+`[#998] <https://github.com/pymor/pymor/pull/998>`_.
+
+
+pyMOR tutorial collection
+~~~~~~~~~~~~~~~~~~~~~~~~~
+Hands-on tutorials provide a good opportunity for new users to get started with
+a software library. In this release a variety of tutorials have been added which
+introduce important pyMOR concepts and basic model order reduction methods. In
+particular users can now learn about:
+
+- :ref:`tutorial01`.
+- :ref:`tutorial-rb`
+- :ref:`tutorial_bt`
+- :ref:`tutorial_mor_with_anns`
+- :ref:`tutorial_external_solver`
```

Comment:
```suggestion
- :doc:`tutorial_builtin_discretizer`.
- :doc:`tutorial_basis_generation`
- :doc:`tutorial_bt`
- :doc:`tutorial_mor_with_anns`
- :doc:`tutorial_external_solver`
```

I believe :ref: does not work.

Reply:
@lbalicki, it seems that your changes in e2e60cd5006e2220f39d869073c97b651a1ff1c3 are somehow based on older versions of the tutorials ... We should probably revert that commit.

Reply:
Sorry that was wrong. It's just that this PR is a bit behind master.